pkg/oc/oc.go

```diff
@@ -133,3 +137,11 @@ func EnableConsolePlugin() {
 	patch_data := "{\"spec\":{\"plugins\":[\"" + strings.Join(plugins, "\",\"") + "\"]}}"
 	cmd.MustSucceed("oc", "patch", "consoles.operator.openshift.io", "cluster", "-p", patch_data, "--type=merge").Stdout()
 }
+
+func GetSecretsData(secretName, namespace string) string {
+	return cmd.MustSucceed("oc", "get", "secrets", secretName, "-n", namespace, "-o", "jsonpath=\"{.data}\"").Stdout()
+}
+
+func CreateChainsImageRegistrySecret(dockerConfig string) {
+	cmd.MustSucceed("oc", "create", "secret", "generic", "chains-image-registry-credentials", "--from-literal=.dockerconfigjson="+dockerConfig, "--from-literal=config.json="+dockerConfig, "--type=kubernetes.io/dockerconfigjson")
```

Comment:
I'm OK to merge this but we already create secrets in 6 places, we should create a generic function for that after this is merged